buildSrc/src/main/kotlin/io/embrace/otel/PublishConfig.kt

```diff
@@ -1,15 +1,13 @@
 package io.embrace.otel
 
 import com.vanniktech.maven.publish.MavenPublishBaseExtension
-import com.vanniktech.maven.publish.SonatypeHost
 import org.gradle.api.Project
 
 fun Project.configurePublishing() {
     project.pluginManager.withPlugin("com.vanniktech.maven.publish") {
         val mavenPublishing = project.extensions.getByType(MavenPublishBaseExtension::class.java)
 
         mavenPublishing.apply {
-            publishToMavenCentral(SonatypeHost.CENTRAL_PORTAL)
```

Comment:
this isn't needed?

Reply:
The latest version of the plugin has deprecated the property & set the default to upload to central. I've removed it to get rid of the warning.